### DISCUSSION TITLE
Notification - In your mastered set changes was made

### DISCUSSION THREAD

**Discussion by coolscribble** [RetroAchievements/RAWeb#1091] [Ideas]
Hi, I would like to receive an page msg when there was an revision for game you mastered/started (with logic same as other similar events: If someone comments on an achievement I created:). Obviously to not trigger when achievement is demoted because of an ticket, but otherwise it would be nice getting notification if something changed (I know there are messages on the discord, but it's sometimes not enough)

**Comment by Jamiras** [comment]
See also #692 (show indicator of revision on mastery badge) and #811 (send notification when revision is completed).

**Comment by Jamiras** [comment]
This was implemented in #1446, but currently only sends emails (no "page msg").
